src/orb.yml

```diff
@@ -441,6 +471,20 @@ examples:
             - cypress/run:
                 yarn: true
 
+  custom-directory:
+    description: |
+      Runs all commands in a custom directory.
```

Comment:
ok, everything seems good, except can you describe this example a little bit better, please? Like the use case is monorepo with front-end tests in a subfolder for example ... right?

Reply:
Yes exactly, I've improved the description a bit, do you think that's good enough now?